models/discount-code/src/discount-code-draft/presets/sample-data-fashion/employee-sale.ts

```diff
@@ -0,0 +1,18 @@
+import { LocalizedString, Reference } from '@commercetools-test-data/commons';
+import * as DiscountCodeDraft from '../..';
+import { TDiscountCodeDraftBuilder } from '../../../types';
+
+const employeeSale = (): TDiscountCodeDraftBuilder =>
+  DiscountCodeDraft.presets
+    .empty()
+    .code('emp15')
+    .name(LocalizedString.presets.empty()['en-US']('Employee Sale'))
+    .description(LocalizedString.presets.empty())
+    // TODO: use the cart discount preset key here when available
```

Comment:
Relevant. A forthcoming PR for cart discounts will use that preset's key in these locations 👍